pkg/install/install.go

```diff
@@ -258,6 +262,16 @@ func Install(ctx context.Context, image string, opts *Options) error {
 	return nil
 }
 
+func validateServiceLBAnnotations(annotations []string) error {
+	for _, annotation := range annotations {
+		split := strings.Split(annotation, "=")
+		if len(split) != 2 {
```

Comment:
`=` should be a valid, supported value in the value part of the annotation.

Reply:
(I think we can just use strings.SplitN)

Reply:
I opted for using `strings.Cut` here.